Makefile

```diff
@@ -25,6 +25,8 @@ serve: build
 lint:
 	$(MAKE) build
 	$(DOCKER_COMPOSE) run --rm app bundle exec govuk-lint-ruby app lib spec Gemfile*
+	$(DOCKER_COMPOSE) run --rm app bundle exec govuk-lint-sass app/assets/stylesheets
```

Comment:
suggest changing to this:

```mk
lint: lint-ruby lint-sass
lint-ruby: build
	$(DOCKER_COMPOSE) run --rm app bundle exec govuk-lint-ruby app lib spec Gemfile*
lint-sass: build
	$(DOCKER_COMPOSE) run --rm app bundle exec govuk-lint-sass app/assets/stylesheets
```

`build` will still only run once.

Reply:
Nice